pkgs/top-level/all-packages.nix

```diff
@@ -5381,6 +5381,11 @@ in
   netcdfcxx4 = callPackage ../development/libraries/netcdf-cxx4 { };
 
   netcdffortran = callPackage ../development/libraries/netcdf-fortran { };
+  
+  netcdffortran-mpi = appendToName "mpi" (netcdffortran.override {
+    hdf5 = hdf5-mpi;
+    netcdf = netcdf-mpi;
+  });
```

Comment:
We haven't done anything with `mpi`, but the direction we went with blas/lapack was to have 1 variable at the top level specifying your variant, and then an overlay can -- in one place -- consistently override everything to be the right flavor.

This ensures you don't end up with two incoherent flavors of something like numpy in your stack.  CC @matthewbauer not sure if we should do something similar for `mpi`.

Reply:
ah, i did get them mixed up. But I think we should have similar conventions between blas and mpi.